### ISSUE TITLE
Plugin not detecting when Airplay is played (to an Airport Express) via an Apple Homepod request

### ISSUE THREAD

**Issue by brettcp** [maxileith/homebridge-airport-express-connected#45]
### Current Situation

I've been using this plugin for a couple of months now and its been working great!  I recently found that the plugin does not detect the Airplay status if the music is initiated via a Siri request on a Homepod.  There are three ways I start music in the house, but only two of them result in this plugin detecting Airplay is playing via my Airport Express.

1.  If I use my phone to play music and then tap on Airplay devices and select an Airplay room.  The plugin detects this properly as "Occupancy Detected" as expected.

2.  If I say to my phone: "Hey Siri, play music in the living room" - The plugin detects this properly as "Occupancy Detected" expected.

3.  If I say to my Apple Homepod mini: "Hey Siri, play music in the living room" - The plugin does not detect music is playing, even though I can open up the Airplay device in Homekit and it looks normal (shows its correct status of Playing with the cover art, controls, etc).  Note this is playing through the Airport Express (and not the Homepod).

Is there any way to get scenario #3 working?

Note: I just did some testing, and scenario #3 does work as expected with the older "[homebridge-airport-express-playing](https://github.com/apexad/homebridge-airport-express-playing)" plugin (which I believe is what this current plugin was forked from).

Thank you so much!

### Proposed Change

Have the plugin detect when Airplay is playing when initiated via a Homepod.

**Comment by maxileith** [comment]
Honestly, I am surprised that you managed to get the old plugin to work with the new home architecture (or did you not update?).

Can you connect to the AirPort Express the third way and then send me some debug logs? No problem if there are a lot of logs. Just make sure to only include the Plugin logs (Config UI > Plugins > Settings Gear of AirPort Express Connected > Plugin Logs.

**Comment by maxileith** [comment]
Can you please check if the issue is solved with version v1.2.3-1?

**Comment by brettcp** [comment]
> Can you please check if the issue is solved with version v1.2.3-1?

Well that was quick!  Yes, v1.2.3-1 seems to have fixed it!  Thank you so much.  I'll do some testing throughout the day to be sure its all working as expected.

**Comment by brettcp** [comment]
> Honestly, I am surprised that you managed to get the old plugin to work with the new home architecture (or did you not update?).
> 
> Can you connect to the AirPort Express the third way and then send me some debug logs? No problem if there are a lot of logs. Just make sure to only include the Plugin logs (Config UI > Plugins > Settings Gear of AirPort Express Connected > Plugin Logs.

The old plugin (homebridge-airport-express-playing) does work, but is definitely slow to respond and also breaks all of my Govee leak detector devices.  Anyway, this appears to be resolved in homebridge-airport-express-connected v1.2.3-1 so I assume you don't need debug logs?

**Comment by maxileith** [comment]
The debug logs are no longer needed. In the previous version I have added an extra condition which has to be met in order for the Airport Express to be reported as connected. It seems like the extra condition is not triggered when a HomePod connects to it.

I added this extra condition since there were certain situations where the AirPort Express was reported as connected unexpectedly. i make sure that this error does not reoccur before publishing this fix in the `latest` tagged version.
